Code/API/Python/tests/test_vecteur.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import annotations
+import unittest
+from funibot_api.funibot import Vecteur
+
+class TestVecteur(unittest.TestCase):
+
+    def test_repr(self):
+        """Test d'un vecteur pour voir s'il se représente bien"""
+        VecteurTest = Vecteur(3,-2,7)
+        self.assertTrue(repr(VecteurTest)== "(3;-2;7)",msg=f"__repr__() donne: {VecteurTest.__repr__()}")
+    
+    def test_add(self):
+        """Test d'addition de deux vecteurs"""
+        VecteurBase = Vecteur(2,7,-4)
+        VecteurAdd = Vecteur(-7,2,10)
+        self.assertTrue(VecteurBase + VecteurAdd == Vecteur(-5,9,6), msg=f"__add__() donne: {VecteurBase + VecteurAdd}")
```

Comment:
Mettre `__repr__()` était relativement legit pour repr, car c'est le seul cas à tester (et même là, mettre ce à quoi on s'attendait pourrait être mieux). Ici, le fait de voir que __add__() donne quelque chose n'aide pas à savoir ce qui a échoué. C'est vrai ailleurs aussi.

```suggestion
        self.assertTrue(VecteurBase + VecteurAdd == Vecteur(-5,9,6), msg=f"Vecteur(2,7,-4) + Vecteur(-7,2,10) donne: {VecteurBase + VecteurAdd}")
```